commit 22100c92ce7ed363dba7a9f5e803165e509a75e2 (refs/changes/79/4879/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-09-26T23:20:02+12:00 (1 year ago)
    
    TRITON-786 dapi needs to learn about "HVM API" from sysinfo

diff --git a/package.json b/package.json
index 32060a9..c73d84f 100644
--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "assert-plus": "1.0.0",
     "async": "1.5.2",
     "bunyan": "1.8.5",
-    "dapi": "git+https://github.com/joyent/sdc-designation.git#af2b13ddc5dbe10a4511936ea46e6655bf6d4aab",
+    "dapi": "git+https://github.com/joyent/sdc-designation.git#master",
     "deep-equal": "1.0.1",
     "dox": "0.9.0",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
